keyboards/inland/keyboard_sw/mk47/info.json

```diff
@@ -0,0 +1,83 @@
+{
+    "keyboard_name": "MK47",
+    "manufacturer": "https://www.microcenter.com",
```

Comment:
The `"manufacturer"` string will go to the `iManufacturer` USB string descriptor; it is unusual to put an URL there — in most cases a plain human readable name is used instead.  (Although apparently Windows does not display this field in any prominent place, in Linux you would see it in the `lsusb` output.)

There is the `"url"` field in JSON, but I'm not sure whether it is actually used anywhere at the moment.